### ISSUE TITLE
VMware Working Group meeting agenda - Sept '17 to Dec 2018

### ISSUE THREAD

**Issue by Akasurde** [ansible/community#233]
The VMware Working Group focuses on delivering modules and features to Ansible to support working with VMware. The group meets every week on Mondays at 16:00 UTC (see [calendar](https://calendar.google.com/calendar/embed?src=ansible.com_pafenslko0e2bqjgujp8f7s0do%40group.calendar.google.com&ctx=UTC)) 

More information related to the VMware Working Group is available [here](https://github.com/ansible/community/tree/master/group-vmware)

**Feel free to add your questions to this ticket to be discussed during the upcoming meeting.**

*(Migrated from the previous agenda at #206, actionable items are available at [VMware: action plan](https://github.com/ansible/community/wiki/VMware:-action-list))*

**Comment by Akasurde** [comment]
[New VMware modules](https://github.com/ansible/ansible/pulls?q=is:open+is:pr+label:vmware+label:new_module) waiting for review/feedback.

- vcenter_folder: https://github.com/ansible/ansible/pull/27005
- vmware_spbm: https://github.com/ansible/ansible/pull/26558
- vmware_host_facts: https://github.com/ansible/ansible/pull/26492
- vmware_cfg_backups: https://github.com/ansible/ansible/pull/21070
- vmware_add_nic: https://github.com/ansible/ansible/pull/19394

Please review these tickets on a weekly basis (if there was new progress, otherwise add instructions to the owner how to continue).

Once merged, please add to [CHANGELOG.md](https://github.com/ansible/ansible/blob/devel/CHANGELOG.md) !

**Comment by Akasurde** [comment]
PR to discuss:
* https://github.com/ansible/ansible/pull/28155
* https://github.com/ansible/ansible/pull/28121

**Comment by Akasurde** [comment]
Meeting ended Mon Sep 11 16:56:49 2017 UTC

- Minutes: https://meetbot.fedoraproject.org/ansible-meeting/2017-09-11/ansible_vmware_working_group_meeting.2017-09-11-16.01.html
- Minutes (text): https://meetbot.fedoraproject.org/ansible-meeting/2017-09-11/ansible_vmware_working_group_meeting.2017-09-11-16.01.txt
- Log: https://meetbot.fedoraproject.org/ansible-meeting/2017-09-11/ansible_vmware_working_group_meeting.2017-09-11-16.01.log.html

**Comment by Akasurde** [comment]
Meeting ended Mon Sep 18 17:15:44 2017 UTC

- Minutes: https://meetbot.fedoraproject.org/ansible-vmware/2017-09-18/ansible_vmware_working_group_meeting.2017-09-18-16.00.html
- Minutes (text): https://meetbot.fedoraproject.org/ansible-vmware/2017-09-18/ansible_vmware_working_group_meeting.2017-09-18-16.00.txt
- Log: https://meetbot.fedoraproject.org/ansible-vmware/2017-09-18/ansible_vmware_working_group_meeting.2017-09-18-16.00.log.html

**Comment by Akasurde** [comment]
Meeting ended Mon Oct  2 17:05:41 2017 UTC.
* Minutes: https://meetbot.fedoraproject.org/ansible-vmware/2017-10-02/ansible_vmware_working_group_meeting.2017-10-02-16.00.html
* Minutes (text): https://meetbot.fedoraproject.org/ansible-vmware/2017-10-02/ansible_vmware_working_group_meeting.2017-10-02-16.00.txt
* Log: https://meetbot.fedoraproject.org/ansible-vmware/2017-10-02/ansible_vmware_working_group_meeting.2017-10-02-16.00.log.html

**Comment by dagwieers** [comment]
I'd like to discuss namespacing VMware modules. Currently almost every VMware-related module is namespaced *vca_* or *vmware_*. We do have *vcenter_* and *vsphere_* already in use.Whereas, Ansible modules are usually named after the technology/product and the managed object. Most of the modules however manage stuff on vCenter (and only vCenter).

[VMware's collection of modules](https://github.com/vmware/ansible-modules-extras-gpl3) correctly prefixes them *vcenter_* and I think we should do this as well. We also have some legacy with **vsphere_copy** which preferably becomes **vcenter_copy**.

So my proposal is:
- Use *vcenter_* for vCenter-only modules
- Keep using *vca_* as we already do for VCA-related modules
- Keep using *vmware_* for everything that covers multiple technologies, like **vmware_guest** for both vCenter and ESXi

Note sure if we have ESXi-related modules, if so we could introduce *esxi_* as a prefix.

**Comment by garbled1** [comment]
I'm very interested in helping out in general in the vmware space.  I'm using ansible quite heavily to manage hosts in vmware/vcenter, and automate builds, and I've needed to add quite a few features/modules to do so.

However, while looking at the code, I've noticed that there is alot of room for general cleanup, which you have also noted in some of the wiki pages for the vmware group.  One thing I would really like to help with, but have no idea how to structure pullups for is the module_utils code.

For example, lets say that 3 modules do a thing, so I want to move it into the module_utils/vmware.py code.  Do I submit a single PR with all 4 changes?  1 PR for module utils and then wait until it's pulled to add the rest?  What if just 3 modules duplicate code from module_utils?  Do I submit 3 separate PR's?

I feel kinda stumped by parts of this.  I would *love* to do that kind of code cleanup/consolidation in the tree, but kind of have no idea how to submit it in a way that isn't difficult for you to process.

**Comment by dagwieers** [comment]
@garbled1 Probably the best way is to discuss your ideas upfront on channel #ansible-vmware (possibly during the weekly meeting). In most cases you can do all changes in a single PR, as long as the result is that everything still works as before :-) Adding unit or integration tests for the stuff you're modifying (if it doesn't exist yet) is a big plus as it will demonstrate everything still works as before.

**Comment by Akasurde** [comment]
* Meeting ended Mon Oct  9 17:04:47 2017 UTC
* Minutes: https://meetbot.fedoraproject.org/ansible-vmware/2017-10-09/ansible_vmware_working_group_meeting.2017-10-09-16.01.html
* Minutes (text): https://meetbot.fedoraproject.org/ansible-vmware/2017-10-09/ansible_vmware_working_group_meeting.2017-10-09-16.01.txt
* Log: https://meetbot.fedoraproject.org/ansible-vmware/2017-10-09/ansible_vmware_working_group_meeting.2017-10-09-16.01.log.html

**Comment by jjahns** [comment]
This happened outside of my window to join. I did talk with the VMW folks on their modules, and there's a lot of things left to do - they did not follow a specific methodology in naming their modules.

**Comment by jjahns** [comment]
@dagwieers @garbled1 @Akasurde 

We had a discussion in IRC today about naming convention, and VMW team was in on the conversation. There is a consensus that moving forward, all modules that hit either esxi OR vcenter and use the pyvmomi library be prefixed as "vmware_" and any existing modules that use the pyvsphere library be prefixed with "vsphere_".  Any additional modules that reflect other products (examples: vcd, vio, vra) be prefixed as "<product>_".

There is an outstanding item that we need to discuss next week, and that is how these modules will be tested in the future. VMW will discuss internally to determine best course of action, but this needs to be the primary topic of next week.

I have the log of the IRC discussion, but I don't think an official meeting was recorded at https://gist.github.com/jjahns/216fe2662da5fecd6267bd68a3777773. Please ding me appropriately if we didn't follow proper procedure (I was unaware of procedure).

Everyone on the IRC appeared to have consensus, so this should be the way forward.

**Comment by kbroughton** [comment]
Is there a good space for troubleshooting / sharing running ansibe at vmware?
I'm trying to hook up our alerting system to send and email to jira to auto-create issues.

I created a service account 'svc...' and added it to jira.  I can send test emails to svc from my vmware email and it creates a ticket.   The following snippet works to send emails to my gmail.com accounts, but it fails without error sending to @vmware.com

       - name: send an email to ops to trigger a jira ticket, assign to cc
      mail:
        host: smtp.mailgun.org
        port: 587
        username: <my username>
        password: "{{ (lookup('credstash', <cred handle>, region='us-west-2') | from_json)['password'] }}"
        to: kesten@wavefront.com
        from: postmaster@wavefront.com
        cc: svc.wf-jira@vmware.com
        subject: gc is thrashing on {{ inventory_hostname }} for {{ service }}
        body: |
          '{{ service }}  has been restarted - version {{ reg_service_version.stdout }} on {{ inventory_hostname }}'
          {{ alertBody.alert.event.annotations.details }}
          autoremediation generated by ansible/tasks/nurse-gc-is-thrashing.yml
          jenkins job Ops/Nurse/Gc-is-thrashing
          attach: /tmp{{ reg_tempfile.path }}/{{ inventory_hostname }}{{ reg_tempfile.path }}
      delegate_to: localhost
 
Has anybody set up jira ticket autocreation?

**Comment by pdellaert** [comment]
Added functionality to vmware_dvs_portgroup PR: ansible/ansible#32298

**Comment by Akasurde** [comment]
Meeting ended Mon Oct 30 16:35:16 2017 UTC.

* Minutes: https://meetbot.fedoraproject.org/ansible-vmware/2017-10-30/ansible_vmware_working_group_meeting.2017-10-30-16.00.html
* Minutes (text): https://meetbot.fedoraproject.org/ansible-vmware/2017-10-30/ansible_vmware_working_group_meeting.2017-10-30-16.00.txt
* Log: https://meetbot.fedoraproject.org/ansible-vmware/2017-10-30/ansible_vmware_working_group_meeting.2017-10-30-16.00.log.html

**Comment by Akasurde** [comment]
Meeting ended Mon Nov  6 17:05:41 2017 UTC
* Minutes: https://meetbot.fedoraproject.org/ansible-vmware/2017-11-06/ansible_vmware_working_group_meeting.2017-11-06-16.00.html
* Minutes (text): https://meetbot.fedoraproject.org/ansible-vmware/2017-11-06/ansible_vmware_working_group_meeting.2017-11-06-16.00.txt
* Log: https://meetbot.fedoraproject.org/ansible-vmware/2017-11-06/ansible_vmware_working_group_meeting.2017-11-06-16.00.log.html

**Comment by Akasurde** [comment]
vmware_guest is not idempotent if 'folder' is not specified 

https://github.com/ansible/ansible/pull/32960

I would like to discuss - what are the adverse effects of this change ?

**Comment by Akasurde** [comment]
PR to discuss - 20 Nov 2017

- https://github.com/ansible/ansible/pull/32885 - Add fix for setting manual MAC address

**Comment by Akasurde** [comment]
Meeting ended Mon Nov 20 16:58:04 2017 UTC.

* Minutes: https://meetbot.fedoraproject.org/ansible-vmware/2017-11-20/ansible_vmware_working_group_meeting.2017-11-20-16.00.html
* Minutes (text): https://meetbot.fedoraproject.org/ansible-vmware/2017-11-20/ansible_vmware_working_group_meeting.2017-11-20-16.00.txt
* Log: https://meetbot.fedoraproject.org/ansible-vmware/2017-11-20/ansible_vmware_working_group_meeting.2017-11-20-16.00.log.html

**Comment by Akasurde** [comment]
I would like to discuss - 

* https://github.com/ansible/ansible/pull/33315

**Comment by Akasurde** [comment]
Meeting ended Mon Dec 11 17:08:28 2017 UTC.

* Minutes: https://meetbot.fedoraproject.org/ansible-vmware/2017-12-11/ansible_vmware_working_group_meeting.2017-12-11-16.00.html
* Minutes (text): https://meetbot.fedoraproject.org/ansible-vmware/2017-12-11/ansible_vmware_working_group_meeting.2017-12-11-16.00.txt
* Log: https://meetbot.fedoraproject.org/ansible-vmware/2017-12-11/ansible_vmware_working_group_meeting.2017-12-11-16.00.log.html

**Comment by Akasurde** [comment]
Meeting ended Mon Dec 18 16:45:19 2017 UTC. 

* Minutes: https://meetbot.fedoraproject.org/ansible-vmware/2017-12-18/ansible_vmware_working_group_meeting.2017-12-18-16.01.html
* Minutes (text): https://meetbot.fedoraproject.org/ansible-vmware/2017-12-18/ansible_vmware_working_group_meeting.2017-12-18-16.01.txt
* Log: https://meetbot.fedoraproject.org/ansible-vmware/2017-12-18/ansible_vmware_working_group_meeting.2017-12-18-16.01.log.html

**Comment by gundalow** [comment]
The 2.5 freeze dates (merge deadlines) have been pushed out by a week.

Please see updated dates 
https://github.com/ansible/ansible/blob/devel/docs/docsite/rst/roadmap/ROADMAP_2_5.rst

To see whats change please see https://github.com/ansible/ansible/commit/1826c27e6841eb313dc3a05dc7f19c96e42c7fbf

**Comment by Akasurde** [comment]
Meeting ended Mon Jan 15 16:28:02 2018 UTC.

* Minutes: https://meetbot.fedoraproject.org/ansible-vmware/2018-01-15/ansible_vmware_working_group_meeting.2018-01-15-16.00.html
* Minutes (text): https://meetbot.fedoraproject.org/ansible-vmware/2018-01-15/ansible_vmware_working_group_meeting.2018-01-15-16.00.txt
* Log: https://meetbot.fedoraproject.org/ansible-vmware/2018-01-15/ansible_vmware_working_group_meeting.2018-01-15-16.00.log.html

**Comment by Akasurde** [comment]
Meeting ended Mon Jan 29 16:17:59 2018 UTC.
* Minutes: https://meetbot.fedoraproject.org/ansible-vmware/2018-01-29/ansible_vmware_working_group_meeting.2018-01-29-16.00.html
* Minutes (text): https://meetbot.fedoraproject.org/ansible-vmware/2018-01-29/ansible_vmware_working_group_meeting.2018-01-29-16.00.txt
* Log: https://meetbot.fedoraproject.org/ansible-vmware/2018-01-29/ansible_vmware_working_group_meeting.2018-01-29-16.00.log.html

**Comment by Akasurde** [comment]
Meeting ended Mon Feb  5 17:22:16 2018 UTC.

* Minutes: https://meetbot.fedoraproject.org/ansible-vmware/2018-02-05/ansible_vmware_working_group_meeting.2018-02-05-15.59.html
* Minutes (text): https://meetbot.fedoraproject.org/ansible-vmware/2018-02-05/ansible_vmware_working_group_meeting.2018-02-05-15.59.txt
* Log: https://meetbot.fedoraproject.org/ansible-vmware/2018-02-05/ansible_vmware_working_group_meeting.2018-02-05-15.59.log.html

**Comment by Akasurde** [comment]
Meeting ended Mon Feb 12 17:11:07 2018 UTC.

* Minutes: https://meetbot.fedoraproject.org/ansible-vmware/2018-02-12/ansible_vmware_working_group_meeting.2018-02-12-16.00.html
* Minutes (text): https://meetbot.fedoraproject.org/ansible-vmware/2018-02-12/ansible_vmware_working_group_meeting.2018-02-12-16.00.txt
* Log: https://meetbot.fedoraproject.org/ansible-vmware/2018-02-12/ansible_vmware_working_group_meeting.2018-02-12-16.00.log.html

**Comment by Akasurde** [comment]
Meeting ended Mon Feb 19 16:29:11 2018 UTC

* Minutes: https://meetbot.fedoraproject.org/ansible-vmware/2018-02-19/ansible_vmware_working_group_meeting.2018-02-19-15.59.html
* Minutes (text): https://meetbot.fedoraproject.org/ansible-vmware/2018-02-19/ansible_vmware_working_group_meeting.2018-02-19-15.59.txt
* Log: https://meetbot.fedoraproject.org/ansible-vmware/2018-02-19/ansible_vmware_working_group_meeting.2018-02-19-15.59.log.html

**Comment by Akasurde** [comment]
Meeting ended Mon Feb 26 16:56:40 2018 UTC

* Minutes: https://meetbot.fedoraproject.org/ansible-vmware/2018-02-26/ansible_vmware_working_group_meeting.2018-02-26-16.01.html
* Minutes (text): https://meetbot.fedoraproject.org/ansible-vmware/2018-02-26/ansible_vmware_working_group_meeting.2018-02-26-16.01.txt
* Log: https://meetbot.fedoraproject.org/ansible-vmware/2018-02-26/ansible_vmware_working_group_meeting.2018-02-26-16.01.log.html

**Comment by Akasurde** [comment]
Meeting ended Mon Mar  5 17:07:16 2018 UTC
* Minutes: https://meetbot.fedoraproject.org/ansible-vmware/2018-03-05/ansible_vmware_working_group_meeting.2018-03-05-16.01.html
* Minutes (text): https://meetbot.fedoraproject.org/ansible-vmware/2018-03-05/ansible_vmware_working_group_meeting.2018-03-05-16.01.txt
* Log: https://meetbot.fedoraproject.org/ansible-vmware/2018-03-05/ansible_vmware_working_group_meeting.2018-03-05-16.01.log.html

**Comment by Akasurde** [comment]
Meeting ended Mon Mar 12 17:09:03 2018 UTC

* Minutes: https://meetbot.fedoraproject.org/ansible-vmware/2018-03-12/ansible_vmware_working_group_meeting.2018-03-12-16.00.html
* Minutes (text): https://meetbot.fedoraproject.org/ansible-vmware/2018-03-12/ansible_vmware_working_group_meeting.2018-03-12-16.00.txt
* Log: https://meetbot.fedoraproject.org/ansible-vmware/2018-03-12/ansible_vmware_working_group_meeting.2018-03-12-16.00.log.html

**Comment by Akasurde** [comment]
Meeting ended Mon Mar 26 17:03:31 2018 UTC

* Minutes: https://meetbot.fedoraproject.org/ansible-vmware/2018-03-26/ansible_vmware_working_group_meeting.2018-03-26-16.00.html
 * Minutes (text): https://meetbot.fedoraproject.org/ansible-vmware/2018-03-26/ansible_vmware_working_group_meeting.2018-03-26-16.00.txt
 * Log: https://meetbot.fedoraproject.org/ansible-vmware/2018-03-26/ansible_vmware_working_group_meeting.2018-03-26-16.00.log.html

**Comment by Akasurde** [comment]
Meeting ended Mon Apr  2 16:56:39 2018 UTC

* Minutes: https://meetbot.fedoraproject.org/ansible-vmware/2018-04-02/ansible_vmware_working_group_meeting.2018-04-02-16.00.html
* Minutes (text): https://meetbot.fedoraproject.org/ansible-vmware/2018-04-02/ansible_vmware_working_group_meeting.2018-04-02-16.00.txt
* Log: https://meetbot.fedoraproject.org/ansible-vmware/2018-04-02/ansible_vmware_working_group_meeting.2018-04-02-16.00.log.html

**Comment by Akasurde** [comment]
Meeting ended Mon Apr 16 17:27:01 2018 UTC

* Minutes: https://meetbot.fedoraproject.org/ansible-vmware/2018-04-16/ansible_vmware_working_group_meeting.2018-04-16-16.20.html
* Minutes (text): https://meetbot.fedoraproject.org/ansible-vmware/2018-04-16/ansible_vmware_working_group_meeting.2018-04-16-16.20.txt
* Log: https://meetbot.fedoraproject.org/ansible-vmware/2018-04-16/ansible_vmware_working_group_meeting.2018-04-16-16.20.log.html

**Comment by Akasurde** [comment]
PR to discuss - 

* https://github.com/ansible/ansible/pull/39339 - disk facts in vmware_guest_facts
* https://github.com/ansible/ansible/pull/33338 - Multiple disks

**Comment by Akasurde** [comment]
Meeting ended Mon May 14 17:33:56 2018 UTC

* Minutes: https://meetbot.fedoraproject.org/ansible-vmware/2018-05-14/ansible_vmware_working_group_meeting.2018-05-14-16.01.html
* Minutes (text): https://meetbot.fedoraproject.org/ansible-vmware/2018-05-14/ansible_vmware_working_group_meeting.2018-05-14-16.01.txt
* Log: https://meetbot.fedoraproject.org/ansible-vmware/2018-05-14/ansible_vmware_working_group_meeting.2018-05-14-16.01.log.html

**Comment by Akasurde** [comment]
Topic to discuss - 

* https://github.com/ansible/ansible/pull/38005#issuecomment-390457211

**Comment by Akasurde** [comment]
No discussion happened.

Meeting ended Mon May 28 16:13:18 2018 UTC

* Minutes: https://meetbot.fedoraproject.org/ansible-vmware/2018-05-28/ansible_vmware_working_group_meeting.2018-05-28-15.59.html
* Minutes (text): https://meetbot.fedoraproject.org/ansible-vmware/2018-05-28/ansible_vmware_working_group_meeting.2018-05-28-15.59.txt
* Log: https://meetbot.fedoraproject.org/ansible-vmware/2018-05-28/ansible_vmware_working_group_meeting.2018-05-28-15.59.log.html

**Comment by Akasurde** [comment]
Meeting ended Mon Jun  4 17:08:29 2018 UTC

 * Minutes: https://meetbot.fedoraproject.org/ansible-vmware/2018-06-04/ansible_vmware_working_group_meeting.2018-06-04-16.00.html
 * Minutes (text): https://meetbot.fedoraproject.org/ansible-vmware/2018-06-04/ansible_vmware_working_group_meeting.2018-06-04-16.00.txt
 * Log: https://meetbot.fedoraproject.org/ansible-vmware/2018-06-04/ansible_vmware_working_group_meeting.2018-06-04-16.00.log.html

**Comment by Akasurde** [comment]
Meeting ended Mon Jun 11 17:06:49 2018 UTC

 * Minutes: https://meetbot.fedoraproject.org/ansible-vmware/2018-06-11/ansible_vmware_working_group_meeting.2018-06-11-16.00.html
 * Minutes (text): https://meetbot.fedoraproject.org/ansible-vmware/2018-06-11/ansible_vmware_working_group_meeting.2018-06-11-16.00.txt
 * Log: https://meetbot.fedoraproject.org/ansible-vmware/2018-06-11/ansible_vmware_working_group_meeting.2018-06-11-16.00.log.html

**Comment by Akasurde** [comment]
Meeting ended Mon Jun 18 16:38:55 2018 UTC

 * Minutes: https://meetbot.fedoraproject.org/ansible-vmware/2018-06-18/ansible_vmware_working_group_meeting.2018-06-18-15.59.html
 * Minutes (text): https://meetbot.fedoraproject.org/ansible-vmware/2018-06-18/ansible_vmware_working_group_meeting.2018-06-18-15.59.txt
 * Log: https://meetbot.fedoraproject.org/ansible-vmware/2018-06-18/ansible_vmware_working_group_meeting.2018-06-18-15.59.log.html

**Comment by Akasurde** [comment]
Meeting ended Mon Jun 25 17:01:57 2018 UTC

 * Minutes: https://meetbot.fedoraproject.org/ansible-vmware/2018-06-25/ansible_vmware_working_group_meeting.2018-06-25-16.01.html
 * Minutes (text): https://meetbot.fedoraproject.org/ansible-vmware/2018-06-25/ansible_vmware_working_group_meeting.2018-06-25-16.01.txt
 * Log: https://meetbot.fedoraproject.org/ansible-vmware/2018-06-25/ansible_vmware_working_group_meeting.2018-06-25-16.01.log.html

**Comment by Akasurde** [comment]
PR to discuss - 

- [ ] https://github.com/ansible/ansible/pull/40521 - VMware: Fix mark as virtual machine method 
- [ ] https://github.com/ansible/ansible/pull/40456 - VMware: additional VSAN facts about Hostsystem
- [ ] https://github.com/ansible/ansible/pull/4082 - VMware: confine search for network till datacenter level 
- [ ] https://github.com/ansible/ansible/pull/39945 - VMware: new module: vmware_category
- [ ] https://github.com/ansible/ansible/pull/39894 - VMware: new module: vmware_category_facts
- [ ] https://github.com/ansible/ansible/pull/38588 - VMware: new module: vmware_guest_customization_facts
- [ ] https://github.com/ansible/ansible/pull/38583 - VMware: new module: vmware_guest_customization

**Comment by Akasurde** [comment]
Meeting ended Mon Jul  2 17:06:30 2018 UTC

 * Minutes: https://meetbot.fedoraproject.org/ansible-vmware/2018-07-02/ansible_vmware_working_group_meeting.2018-07-02-16.05.html
 * Minutes (text): https://meetbot.fedoraproject.org/ansible-vmware/2018-07-02/ansible_vmware_working_group_meeting.2018-07-02-16.05.txt
 * Log: https://meetbot.fedoraproject.org/ansible-vmware/2018-07-02/ansible_vmware_working_group_meeting.2018-07-02-16.05.log.html

**Comment by Akasurde** [comment]
Meeting ended Mon Jul  9 17:04:38 2018 UTC

 * Minutes: https://meetbot.fedoraproject.org/ansible-vmware/2018-07-09/ansible_vmware_working_group_meeting.2018-07-09-16.01.html
 * Minutes (text): https://meetbot.fedoraproject.org/ansible-vmware/2018-07-09/ansible_vmware_working_group_meeting.2018-07-09-16.01.txt
 * Log: https://meetbot.fedoraproject.org/ansible-vmware/2018-07-09/ansible_vmware_working_group_meeting.2018-07-09-16.01.log.html

**Comment by Akasurde** [comment]
Meeting ended Mon Jul 16 16:42:39 2018 UTC

 * Minutes: https://meetbot.fedoraproject.org/ansible-vmware/2018-07-16/ansible_vmware_working_group_meeting.2018-07-16-16.01.html
 * Minutes (text): https://meetbot.fedoraproject.org/ansible-vmware/2018-07-16/ansible_vmware_working_group_meeting.2018-07-16-16.01.txt
 * Log: https://meetbot.fedoraproject.org/ansible-vmware/2018-07-16/ansible_vmware_working_group_meeting.2018-07-16-16.01.log.html

**Comment by Akasurde** [comment]
Meeting ended Mon Jul 23 16:59:04 2018 UTC

 * Minutes: https://meetbot.fedoraproject.org/ansible-vmware/2018-07-23/ansible_vmware_working_group_meeting.2018-07-23-16.02.html
 * Minutes (text): https://meetbot.fedoraproject.org/ansible-vmware/2018-07-23/ansible_vmware_working_group_meeting.2018-07-23-16.02.txt
 * Log: https://meetbot.fedoraproject.org/ansible-vmware/2018-07-23/ansible_vmware_working_group_meeting.2018-07-23-16.02.log.html

**Comment by dagwieers** [comment]
PR to discuss:
- https://github.com/ansible/ansible/pull/43426

**Comment by Im0** [comment]
Did you guys end up agreeing to collaborate on a wiki first?  If so, whereabouts?  Ta.

**Comment by Akasurde** [comment]
@Im0 There was no agreement made as such but you can find the wiki here - https://github.com/ansible/community/wiki/VMware. Also, feel free to fill gaps in documentation found at https://docs.ansible.com/ansible/devel/vmware/index.html

**Comment by gundalow** [comment]
## Ansible Contributor Summit 6 (Part of AnsibleFest 2018 Austin, Texas)

As part of http://www.ansible.com/ansiblefest

* AnsibleFest Austin will be on both Tuesday 2nd and Wednesday 3rd October 2018
* The Contributor Summit will also be over two days, on Monday 1st and Thursday 4th October, from 9:30am to 4pm (time subject to change).

### This is your event

Ansible succeeds because of its community, and this contributor summit is for you. If you have any topics to discuss, large or small, please be sure to post those topics on the etherpad. For topics that other had added please vote by adding +1

https://etherpad.openstack.org/p/ansible-summit-october-2018 

Please see https://groups.google.com/forum/#!topic/ansible-devel/ivrpVbLya8g for further info

### Attending in person

We realize that not everybody can attend in person, which is why we’ll again be arranging a video conference and a meeting time on IRC in #ansible-meeting to encourage remote participation. This has worked very well for the past Contributor Summits. We’ll be sending out an email with the information for remote participation about a week before the Contributor Summit. 

If you want to attend in-person please sign up at: https://www.eventbrite.com/e/ansible-contributor-conference-austin-2018-tickets-48557063484 

After signing up you'll receive an email from Eventbrite with a discount code for AnsibleFest Austin. 


### Attending remotely

Please add your name at the bottom of this etherpad: https://etherpad.openstack.org/p/ansible-summit-october-2018 

### Call to action

1) Add ideas to https://etherpad.openstack.org/p/ansible-summit-october-2018 
2) Attending in person: https://www.eventbrite.com/e/ansible-contributor-conference-austin-2018-tickets-48557063484 then use discount code to sign up to AnsibleFest
3) Attending in person OR virtually: Add your name to the end of https://etherpad.openstack.org/p/ansible-summit-october-2018 
4) Happy automating!

### Questions?
Thanks for all of your contributions to Ansible, and we hope to see you in Austin! And if you have any questions at all, please let us know. 

Speak to `gundalow` on Freenode, or anyone else in `#ansible-devel`

**Comment by Akasurde** [comment]
* Meeting ended Mon Aug 13 16:46:04 2018 UTC
 * Minutes: https://meetbot.fedoraproject.org/ansible-vmware/2018-08-13/ansible_vmware_working_group_meeting.2018-08-13-16.01.html
 * Minutes (text): https://meetbot.fedoraproject.org/ansible-vmware/2018-08-13/ansible_vmware_working_group_meeting.2018-08-13-16.01.txt
 * Log: https://meetbot.fedoraproject.org/ansible-vmware/2018-08-13/ansible_vmware_working_group_meeting.2018-08-13-16.01.log.html

**Comment by Akasurde** [comment]
* Meeting ended Mon Aug 20 17:01:07 2018 UTC

 * Minutes: https://meetbot.fedoraproject.org/ansible-vmware/2018-08-20/ansible_vmware_working_group_meeting.2018-08-20-16.00.html
 * Minutes (text): https://meetbot.fedoraproject.org/ansible-vmware/2018-08-20/ansible_vmware_working_group_meeting.2018-08-20-16.00.txt
 * Log: https://meetbot.fedoraproject.org/ansible-vmware/2018-08-20/ansible_vmware_working_group_meeting.2018-08-20-16.00.log.html

**Comment by Tomorrow9** [comment]
PR to discuss:
- [ansible/ansible/#42995](https://github.com/ansible/ansible/issues/42995)
- [ansible/ansible/#43000](https://github.com/ansible/ansible/issues/43000)

**Comment by stroobl** [comment]
PR to discuss:
 * https://github.com/ansible/ansible/pull/39792

**Comment by Akasurde** [comment]
No discussion

* Meeting ended Mon Sep 10 16:57:23 2018 UTC

 * Minutes: https://meetbot.fedoraproject.org/ansible-vmware/2018-09-10/ansible_vmware_working_group_meeting.2018-09-10-16.00.html
 * Minutes (text): https://meetbot.fedoraproject.org/ansible-vmware/2018-09-10/ansible_vmware_working_group_meeting.2018-09-10-16.00.txt
 * Log: https://meetbot.fedoraproject.org/ansible-vmware/2018-09-10/ansible_vmware_working_group_meeting.2018-09-10-16.00.log.html

**Comment by agowa338** [comment]
PR to discuss:
ansible/ansible#46625

**Comment by Akasurde** [comment]
Meeting ended Mon Oct  8 17:11:11 2018 UTC

 * Minutes: https://meetbot.fedoraproject.org/ansible-vmware/2018-10-08/ansible_vmware_working_group_meeting.2018-10-08-16.02.html
 * Minutes (text): https://meetbot.fedoraproject.org/ansible-vmware/2018-10-08/ansible_vmware_working_group_meeting.2018-10-08-16.02.txt
 * Log: https://meetbot.fedoraproject.org/ansible-vmware/2018-10-08/ansible_vmware_working_group_meeting.2018-10-08-16.02.log.html

**Comment by dagwieers** [comment]
There is a need to have more consistent module parameters for the VMware modules as discussed here: https://github.com/ansible/ansible/pull/46225#discussion_r223464363
And we can leave aliases for the original parameter names.

**Comment by Akasurde** [comment]
* Meeting ended Mon Oct 15 17:28:15 2018 UTC
* Minutes: https://meetbot.fedoraproject.org/ansible-vmware/2018-10-15/ansible_vmware_working_group_meeting.2018-10-15-16.03.html
* Minutes (text): https://meetbot.fedoraproject.org/ansible-vmware/2018-10-15/ansible_vmware_working_group_meeting.2018-10-15-16.03.txt
* Log: https://meetbot.fedoraproject.org/ansible-vmware/2018-10-15/ansible_vmware_working_group_meeting.2018-10-15-16.03.log.html

**Comment by Akasurde** [comment]
* Meeting ended Mon Oct 22 17:09:31 2018 UTC
 * Minutes: https://meetbot.fedoraproject.org/ansible-vmware/2018-10-22/ansible_vmware_working_group_meeting.2018-10-22-16.00.html
 * Minutes (text): https://meetbot.fedoraproject.org/ansible-vmware/2018-10-22/ansible_vmware_working_group_meeting.2018-10-22-16.00.txt
 * Log: https://meetbot.fedoraproject.org/ansible-vmware/2018-10-22/ansible_vmware_working_group_meeting.2018-10-22-16.00.log.html

**Comment by Akasurde** [comment]
* Meeting ended Mon Oct 29 17:17:20 2018 UTC
 * Minutes: https://meetbot.fedoraproject.org/ansible-vmware/2018-10-29/ansible_vmware_working_group_meeting.2018-10-29-16.01.html
 * Minutes (text): https://meetbot.fedoraproject.org/ansible-vmware/2018-10-29/ansible_vmware_working_group_meeting.2018-10-29-16.01.txt
 * Log: https://meetbot.fedoraproject.org/ansible-vmware/2018-10-29/ansible_vmware_working_group_meeting.2018-10-29-16.01.log.html

**Comment by Akasurde** [comment]
* Meeting ended Mon Nov 19 16:57:59 2018 UTC
* Minutes: https://meetbot.fedoraproject.org/ansible-vmware/2018-11-19/ansible_vmware_working_group_meeting.2018-11-19-16.01.html
* Minutes (text): https://meetbot.fedoraproject.org/ansible-vmware/2018-11-19/ansible_vmware_working_group_meeting.2018-11-19-16.01.txt
* Log: https://meetbot.fedoraproject.org/ansible-vmware/2018-11-19/ansible_vmware_working_group_meeting.2018-11-19-16.01.log.html

**Comment by agowa338** [comment]
For the next meeting, can you please give me some feedback on #46625? What is the process for it being merged? Is there anything that needs to be done before it can be merged? I guess so, as it has not been merged yet.

**Comment by Akasurde** [comment]
* Meeting ended Mon Dec  3 17:36:12 2018 UTC
* Minutes: https://meetbot.fedoraproject.org/ansible-vmware/2018-12-03/ansible_vmware_working_group_meeting.2018-12-03-16.02.html
* Minutes (text): https://meetbot.fedoraproject.org/ansible-vmware/2018-12-03/ansible_vmware_working_group_meeting.2018-12-03-16.02.txt
* Log: https://meetbot.fedoraproject.org/ansible-vmware/2018-12-03/ansible_vmware_working_group_meeting.2018-12-03-16.02.log.html

**Comment by Akasurde** [comment]
PR to discuss - 

- https://github.com/ansible/ansible/pull/47962

**Comment by Akasurde** [comment]
No discussion

*  Meeting ended Mon Dec 10 16:52:49 2018 UTC
*  Minutes: https://meetbot.fedoraproject.org/ansible-vmware/2018-12-10/ansible_vmware_working_group_meeting.2018-12-10-16.05.html
*  Minutes (text): https://meetbot.fedoraproject.org/ansible-vmware/2018-12-10/ansible_vmware_working_group_meeting.2018-12-10-16.05.txt
* Log: https://meetbot.fedoraproject.org/ansible-vmware/2018-12-10/ansible_vmware_working_group_meeting.2018-12-10-16.05.log.html
